CONTRIBUTING.md

```diff
@@ -91,8 +91,30 @@ General requirements for submitting rules: don't add rules to the beginning of t
 
 ### AdGuard Annoyances filter
 
-* Purpose: this filter blocks various kinds of annoyances from sites.
-* [Annoyances folder](https://github.com/AdguardTeam/AdguardFilters/tree/master/AnnoyancesFilter/sections)
+* Purpose: Blocks irritating elements on web pages including cookie notices, third-party widgets and in-page pop-ups.
```

Comment:
If I'm not wrong, all other similar sections (these with `Purpose:  [...]`) do not start with capital letter (I mean capital letter in a `Blocks` word).
So, maybe it would be better to be consistent and use a small letter too?

Reply:
I agree with that.
Also after `Purpose:` we have `this filter [...]`